CHANGELOG.md

```diff
@@ -4,7 +4,14 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [0.6.1] - 2020-05-28
+## [0.6.3] - 2020-05-xx
```

Comment:
why this is not under _unreleased_? Have we changed the format?

Reply:
To avoid to change Master later :roll_eyes: